.github/workflows/documentation.yml

```diff
@@ -31,4 +31,4 @@ jobs:
       run: make html
       working-directory: ./doc
     # TODO: Parse warnings
-    # TODO: Deploy to gh-pages without removing existing folders with docs of historical releases
\ No newline at end of file
+    # TODO: Deploy to gh-pages without removing existing folders with docs of historical releases
```

Comment:
We should normally always have newlines at the end of files. Just a good practice.

Reply:
I've been using https://pre-commit.com for quite some time, it helps to automate linting and other mundane tasks.

Reply:
Yep, I've used that at work for a while now too. Would probably be a good thing to add at some point.

Reply:
https://github.com/nanodbc/nanodbc/issues/417